### ISSUE TITLE
Error: App not installed

### ISSUE THREAD

**Issue by JuliannaG8** [CakeKing64/P5IM-android-10-fix#2]
No matter which release I pick, I always get a message saying that the app is not installed, so it doesn't update.

**Comment by JuliannaG8** [comment]
Figured out the fix- you need to first UNINSTALL the version of the app you are using, then reinstall it using one of the releases here.
To play it safe, I installed the releases in order, but I'm sure you could just jump straight to the lates one. Leaving this open so others experiencing the same problem can see this.

**Comment by CakeKing64** [comment]
Yeah only the latest needs to be installed, I think 'll just hide the previous releases seeing as there really isn't any reason to use them (or maybe just leave a notice) :)

**Comment by PhantomTrickster** [comment]
I'm installing the latest version and have no prior version installed and get that same error "App not installed"

Anything I'm doing wrong or could do differently to get it to work? 

Much appreciated

**Comment by CakeKing64** [comment]
Best way to get a proper error code is to install the app using [adb](https://developer.android.com/studio/releases/platform-tools)

You'll need to enable developer mode if you haven't already (https://developer.android.com/studio/command-line/adb#Enabling)

Once you have all of that you can run the command
```
adb install v2.3-android-10-p2.apk
```
And return to me with what code it spat out trying to install it and we can continue from there

**Comment by PhantomTrickster** [comment]
Okay I was finally able to sit down and try it and it did fail and this was the error I received:

adb: failed to install v2.3-android-10-p2.apk: Failure [INSTALL_FAILED_UPDATE_IN
COMPATIBLE: Package com.ngreenan.persona5imapp signatures do not match previousl
y installed version; ignoring!]

**Comment by CakeKing64** [comment]
Ok so, it would appear that the app is already installed or something like that.
But seeing as we have adb ready we might as well use that

try using
```
adb uninstall com.ngreenan.persona5imapp
```
and then reinstall it with the cool kids command
```
adb install v2.3-android-10-p2.apk
```

and hopefully that works if not hand over the new error code :)

**Comment by PhantomTrickster** [comment]
Okay so here's how that went: 

I did the uninstall and it said Success. Ran the install and got the same error message. Restarted my device and tried again and same error.

What's weird is how the error is implying it's still installed. To see what would happen I tried the uninstall again and it wouldn't say success or do anything really. The command prompt line would move down 1 line but wouldn't give success or even an error. I had to close and reopen command prompt to do anything else. 

So it definitely seemed to uninstall the first time but still gives the same error message (pasted below, even though it's the same as far as I can tell but what do I know lol I appreciate your help seriously):

Performing Streamed Install
adb: failed to install v2.3-android-10-p2.apk: Failure [INSTALL_FAILED_UPDATE_IN
COMPATIBLE: Package com.ngreenan.persona5imapp signatures do not match previousl
y installed version; ignoring!]

**Comment by CakeKing64** [comment]
Ok so, i've decided instead of pleading with android to work

here is a very cool modified version that has a different package name so there shouldn't be any previous signatures to match to
(under com.cakeking64.p5imapp_fix_i_hope instead of com.ngreenan.persona5imapp)

Seemed to work perfectly fine (with like 5 minutes of testing with both versions installed)
[ignoring_the_problem.zip](https://github.com/CakeKing64/P5IM-android-10-fix/files/6610717/ignoring_the_problem.zip)

So hope this just kinda bypasses the problem lol
